build_tools/github_actions/runner/gcp/update_instance_groups.py

```diff
@@ -208,12 +192,13 @@ def main(args):
     print(f"Updating {mig.name} to new versions:"
           f" {summarize_versions(new_versions)}")
 
-    migs_client.patch(
-        project=args.project,
-        region=region,
-        instance_group_manager=mig.name,
-        instance_group_manager_resource=compute.InstanceGroupManager(
-            versions=new_versions, update_policy=update_policy))
+    if not args.dry_run:
+      migs_client.patch(
+          project=args.project,
+          region=region,
+          instance_group_manager=mig.name,
+          instance_group_manager_resource=compute.InstanceGroupManager(
+              versions=new_versions, update_policy=update_policy))
     print(f"Successfully updated {mig.name}")
```

Comment:
Seeing "Successfully updated" in the output could be confusing in dry run mode. Maybe print something different for dry run mode?

Reply:
The general principle I've seen for dry runs is that they should print all the same output as the real run. That's kind of the point :-) I could have it print something instead of making this call that reminds people it's a dry run though